plugins/browser-plugin-snowplow-ecommerce/src/types.ts

```diff
@@ -17,7 +17,8 @@ export interface Action {
     | 'promo_view'
     | 'checkout_step'
     | 'transaction'
-    | 'refund';
+    | 'refund'
+    | 'transaction_error';
```

Comment:
Is https://github.com/snowplow/snowplow-rdb-loader/issues/1158 still a problem for e.g. redshift? Do users need to update any column lengths or anything with this update?

Reply:
Did not have a clue about the issue and thanks for mentioning it, will tag someone on the iglu review.